gender_novels/corpus_gen.py

```diff
@@ -310,13 +315,26 @@ def get_subject(author, title, id = None):
     """
     pass
 
-def write_metadata(novel_metadata, path):
+def write_metadata(novel_metadata):
     """
     Writes a row of metadata for a novel into the csv at path
```

Comment:
That's pretty neat!
However, I'm not sure we want a (not yet finished) file to be the result of a doctest. @mscuthbert: Thoughts?
It might be an option (for the doctest) to read the file back, print the first line back, and then delete the file. Not sure here.